### PR TITLE
Setting-envars-docs #3582

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Expanding on options to configure envars in OpenSearch

### Issues Resolved
Partly closes https://github.com/opensearch-project/documentation-website/issues/3582

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by scrawfor99 — APPROVED** [review]

File: _install-and-configure/configuring-opensearch/index.md [hunk 1]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
 
 ## Specifying settings as environment variables
 
+You can specify environment variables in different ways.
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/index.md:28]:
```suggestion
You can specify environment variables in the following ways.
```

File: _install-and-configure/configuring-opensearch/index.md [hunk 2]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
...
 You can specify environment variables as arguments using `-E` when launching OpenSearch:
 
 ```bash
 ./opensearch -Ecluster.name=opensearch-cluster -Enode.name=opensearch-node1 -Ehttp.host=0.0.0.0 -Ediscovery.type=single-node
 ```
 {% include copy.html %}
 
+### Directly in the shell environment
+
+You can configure the environment variables directly in shell before starting OpenSearch:
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/index.md:41]:
```suggestion
You can configure the environment variables directly in a shell environment before starting OpenSearch, as shown in the following example:
```

File: _install-and-configure/configuring-opensearch/index.md [hunk 3]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
...
+```bash
+export OPENSEARCH_JAVA_OPTS="-Xms2g -Xmx2g"
+export OPENSEARCH_PATH_CONF="/etc/opensearch"
+./opensearch
+```
+{% include copy.html %}
+
+### Systemd service file
+
+When running OpenSearch as a service managed by systemd, you can specify environment variables in the service file:
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/index.md:52]:
```suggestion
When running OpenSearch as a service managed by `systemd`, you can specify environment variables in the service file, as shown in the following example:
```

File: _install-and-configure/configuring-opensearch/index.md [hunk 4]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
...
+
+```bash
+# /etc/systemd/system/opensearch.service.d/override.conf
+[Service]
+Environment="OPENSEARCH_JAVA_OPTS=-Xms2g -Xmx2g"
+Environment="OPENSEARCH_PATH_CONF=/etc/opensearch"
+```
+After creating or modifying the file, reload the systemd configuration and restart the service:
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/index.md:60]:
```suggestion
After creating or modifying the file, reload the systemd configuration and restart the service using the following command:
```

File: _install-and-configure/configuring-opensearch/index.md [hunk 5]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
...
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl restart opensearch
+```
+{% include copy.html %}
+
+### Docker environment variables
+
+When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run:
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/index.md:70]:
```suggestion
When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run, as shown in the following command:
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

File: _install-and-configure/configuring-opensearch/index.md [hunk 6]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
 
 ## Specifying settings as environment variables
 
+You can specify environment variables in the following ways.
+
+### Arguments as start up
```
Comment (natebower) [inline _install-and-configure/configuring-opensearch/index.md:30]: Should "as" be "at"? Also, if "startup" is a noun, it should be one word with no space.

File: _install-and-configure/configuring-opensearch/index.md [hunk 7]
```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
...
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl restart opensearch
+```
+{% include copy.html %}
+
+### Docker environment variables
+
+When running OpenSearch in Docker, you can specify environment variables using the -e option with Docker run, as shown in the following command:
```
Comment (natebower) [inline _install-and-configure/configuring-opensearch/index.md:70]: Should it be "the `-e` option with `docker run`"?

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS @AntonEliatra  A few comments for you. Thanks!

**Comment by AntonEliatra** [comment]
@natebower good catch, thats updated now

**Pull request merged**